src/app/Footer.tsx

```diff
@@ -0,0 +1,30 @@
+import Image from 'next/image';
+import styles from '../../styles/app/Footer.module.css';
+
+export const Footer = () => {
+	return (
+		<footer className={styles.footer}>
+			<div className={styles.separator}>
+				<hr />
+				<Image
+					src="/Monogram-white.svg"
```

Comment:
maybe rename this one with shortividio_logo

Reply:
the className ?